generators/bdk/java/templates/build.gradle.ejs

```diff
@@ -1,11 +1,11 @@
 plugins {
     id 'java-library'
-<% if (framework == 'java') { %>
+<% if (framework == 'java') { -%>
```

Comment:
It does the trick of removing empty lines? cool

Reply:
yes it is. It trims the empty lines before the text.